geomesa-fs/geomesa-fs-storage/geomesa-fs-storage-common/src/main/scala/org/locationtech/geomesa/fs/storage/common/partitions/XZ2Scheme.scala

```diff
@@ -13,17 +13,22 @@ import org.locationtech.geomesa.fs.storage.common.partitions.SpatialScheme.Spati
 import org.locationtech.jts.geom.Geometry
 import org.locationtech.sfcurve.IndexRange
 import org.opengis.feature.simple.SimpleFeature
+import org.opengis.filter.Filter
 
 case class XZ2Scheme(bits: Int, geom: String, geomIndex: Int) extends SpatialScheme(bits, geom) {
 
   private val xz2 = XZ2SFC((bits / 2).asInstanceOf[Short])
 
+  override def pattern: String = s"$bits-bit-xz2"
+
   override def getPartitionName(feature: SimpleFeature): String = {
     val geometry = feature.getAttribute(geom).asInstanceOf[Geometry]
     val envelope = geometry.getEnvelopeInternal
     xz2.index(envelope.getMinX, envelope.getMinY, envelope.getMaxX, envelope.getMaxY).formatted(format)
   }
 
+  override def getCoveringFilter(partition: String): Filter = throw new NotImplementedError()
```

Comment:
There's no covering filter for an XZ2 partition?

Reply:
there is but I didn't figure it out...

Reply:
whew... can you add a comment + ticket to cover that?  Maybe some a note in the error?

Reply:
hmm, what note would you like to see? NotImplemented seems pretty clear ;)
created https://geomesa.atlassian.net/browse/GEOMESA-2967 to track the issue

Reply:
Can we split out something a message referencing that ticket? *shrugs*

Reply:
```suggestion
  override def getCoveringFilter(partition: String): Filter =
    throw new NotImplementedError("https://geomesa.atlassian.net/browse/GEOMESA-2967")
```